bloggies_frontend/src/RoutedPages/PendingPage/index.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react' 
```

Comment:
Is the `PendingPage` component used for rendering the page for when the user has just finished the registration flow? If so, I'm wondering if it would be best to have that intent be rendered in the `RegisterStatusPage` component.

Reply:
I think this issue can be resolved once we decide on how the UI will look like for the post-registration status page(s). For now, I think it's difficult to conclude whether we would be needing a separate component for this page or not without it.

Reply:
Oh, wow. It seems like I misunderstood the user flow. I think the `RegisterStatusPage` component has to be renamed to `MembershipEligibilityStatusPage` in this case.

Reply:
> Is the `PendingPage` component used for rendering the page for when the user has just finished the membership form? If so, I'm wondering if it would be best to have that intent be rendered in the `RegisterStatusPage` component.
> 
> EDIT: Oh, wow. It seems like I misunderstood the user flow while making the bare bone page components. I think the `RegisterStatusPage` component has to be renamed to `MembershipEligibilityStatusPage` in this case.

The `PendingPage` component would render after the user finished the form if their membership status is in the pending state.

Reply:
> I think this issue can be resolved once we decide on how the UI will look like for the post-membership eligibility status page(s). For now, I think it's difficult to conclude whether we would be needing a separate component for this page or not without it.

Yes I think you're right maybe we need to do some UI mocking for some of these screens to hash it out more to see what it should look like.